src/localization/localization.ts

```diff
@@ -128,7 +129,19 @@ const localization = {
             join: "Join",
             connected: (count: number) => `Connected to ${pluralize(count, "Zeplin Component")}.`,
             connect: "Connect to Zeplin components",
-            notFound: (name: string) => `No component matching “${name}” in projects/styleguides you’re a member of`,
+            askMigration: "Would you like to migrate zeplinNames on your configuration to zeplinIds?",
+            migrate: "Migrate Zeplin component names to ids",
+            migrationInProgress: "Getting components data for migration…",
+            noBarrelFoundForMigration: "No project or styleguide found on configuration file.",
+            noComponentFoundForMigration: "No component found on configuration file.",
```

Comment:
```suggestion
            noComponentFoundForMigration: "No component found on the configuration file.",
```

Reply:
We didn't use "the" when mentioning the configuration file. see other copies